models/Manage.js

```diff
@@ -26,56 +26,23 @@ module.exports = class Manage {
   }
 
   static create(attributes) {
-    return new Promise((resolve, reject) => {
-      const client = db.createClient();
-      const { sql, values } = db.queryBuilder(Manage, attributes);
-
-      client.connect();
-      client.on('error', (err) => reject(err));
-
-      const query = client.query(sql, values);
+    const { sql, values } = db.queryBuilder(Manage, attributes);
```

Comment:
Note to self: Based on the name alone, I want to check the `buildQuery` helper for SQL injection vulnerabilities.